packages/build/src/plugins_core/secrets_scanning/index.ts

```diff
@@ -52,16 +53,31 @@ const coreStep: CoreStepFunction = async function ({
   if (envVars['SECRETS_SCAN_OMIT_PATHS'] !== undefined) {
     log(logs, `SECRETS_SCAN_OMIT_PATHS override option set to: ${envVars['SECRETS_SCAN_OMIT_PATHS']}\n`)
   }
+  const enhancedScanningEnabledInEnv = isEnhancedSecretsScanningEnabled(envVars)
+  if (enhancedSecretScan && !enhancedScanningEnabledInEnv) {
+    logSecretsScanSkipMessage(
+      logs,
+      'Enhanced secrets detection disabled via ENHANCED_SECRETS_SCAN_ENABLED flag set to false.',
+    )
+  }
+  if (
+    enhancedSecretScan &&
+    !enhancedScanningEnabledInEnv &&
+    envVars['ENHANCED_SECRETS_SCAN_OMIT_VALUES'] !== undefined
+  ) {
+    log(
+      logs,
+      `ENHANCED_SECRETS_SCAN_OMIT_VALUES override option set to: ${envVars['ENHANCED_SECRETS_SCAN_OMIT_VALUES']}\n`,
+    )
+  }
```

Comment:
Is this right condition? `!enhancedScanningEnabledInEnv` would indicate that enhanced scan should not run (?) so logging something about override option seems a bit odd specifically for case when something should not run?

Reply:
Ah yes, you're absolutely right - thank you